packages/terra-slider/src/Slider.module.scss

```diff
@@ -0,0 +1,212 @@
+@import '~terra-mixins/lib/Mixins';
+
+// Themes
+@import './orion-fusion-theme/Slider.module';
+@import './clinical-lowlight-theme/Slider.module';
+
+:local {
+  .slider {
+    align-items: center;
+    display: inline-flex;
+    min-height: 44px;
+  }
+
+  .label {
+    color: var(--terra-slider-label-color, #1c1f21);
+    font-weight: 700;
+    margin-right: 7px;
+    text-align: center;
+    white-space: initial;
+    width: 20%;
+  }
+
+  .min-label {
+    color: #6f7477;
```

Comment:
This makes me think the lowlight theme is not being handled properly.  There should be a theme variable as I would not expect the font color to be the same for lowlight and light themes.  It should also be configurable for other themes.

Reply:
These are infact given same values by UX. I agree with you though. I will double check with them on this and if we need to change them to be different.